recipe/patches/fix-macos-dylib-version.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: conda-forge <conda@conda-forge.org>
+Date: Mon, 2 Dec 2024 10:00:00 +0000
+Subject: [PATCH] Fix macOS dylib version format
+
+Remove 0.0. prefix from LLAMA_INSTALL_VERSION to avoid macOS linker error
+with large build numbers (e.g. 7229).
+
+---
+ CMakeLists.txt | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
There is a reference for this issue: 
https://github.com/ggml-org/llama.cpp/issues/17258

Instead you could keep LLAMA_INSTALL_VERSION as intended and patch this line https://github.com/ggml-org/llama.cpp/blob/bde188d60f58012ada0725c6dd5ba7c69fe4dd87/src/CMakeLists.txt#L140 to something else.